backend/test/index.test.js

```diff
@@ -0,0 +1,51 @@
+const Client = require("socket.io-client");
+const CreateLobbyDTO = require("../domain/DTO/request/CreateLobbyDTO");
+const server = require("../index")
+
+describe("Create-lobby event test", () => {
+    let clientSocket;
+
+    beforeAll((done) => {
+        clientSocket = new Client(`http://localhost:4001`);
+        done();
```

Comment:
Do we not have to connect the client socket?
`clientSocket.on("connect", done);`